### PR TITLE
Fix manifold

### PR DESCRIPTION
**Pull request by luca-heltai**
Addresses #3013.

### CODE REVIEW

File: tests/manifold/projection_manifold_01.cc [hunk 1]
```diff
@@ -0,0 +1,74 @@
+//-------------------------------------------------------------------
+//    Copyright (C) 2016 by the deal.II authors.
```
Comment (luca-heltai) [inline tests/manifold/projection_manifold_01.cc]: I fixed this. It was copied from a local test ... sorry.

Comment (davydden) [inline tests/manifold/projection_manifold_01.cc]: still it looks different from what i quoted. I am not specialist in different flavours of licenses, though.

**Comment by tjhei** [comment]
Are any other tests influenced by this change?

**Comment by luca-heltai** [comment]
I ran all tests under `manifold`, and got 100% passes. Other tests should not be affected.

File: tests/manifold/projection_manifold_01.cc (hunk 1 above)
Comment (luca-heltai) [inline tests/manifold/projection_manifold_01.cc]: It uses the copyright of the year when it was created...

Comment (luca-heltai) [inline tests/manifold/projection_manifold_01.cc]: I found both versions of the text in the tests directory... If you think I should change it, I will...

Comment (davydden) [inline tests/manifold/projection_manifold_01.cc]: sure, i am not speaking about that. The content is different to what i quoted above (unless there are some glitches in GitHub which does not show the current version).

Comment (davydden) [inline tests/manifold/projection_manifold_01.cc]: i was just curios why there are different headers at all. One states that the file is part of `deal.II`, another does not. Both refer to `LGPL` but the text is different.

**Comment by luca-heltai** [comment]
Let me know if you want me to change the header of the test file. Otherwise this is ready to merge.

**Comment by tjhei** [comment]
> Let me know if you want me to change the header of the test file.

Yes, please do.

**Pull request merged**